adage/analyze/management/commands/import_node_gene_network.py

```diff
@@ -123,17 +141,26 @@ def check_and_import(file_handle, ml_model):
                 continue
 
             # Raise an exception if the combination of (node, gene)
-            # already exists in Particapation table.  Instead of relying
+            # already exists in Participation table. Instead of relying
```

Comment:
Ah gotcha - good catch! I was going for the style of writing in English that just uses one space after a sentence-ending period, but I'm glad you pointed out that PEP8 states otherwise! I will change this.